rebar.config

```diff
@@ -23,7 +23,7 @@
     [{test,
         [{deps,
             [ {meck, "0.9.2"}
-            , {emqx, {git_subdir, "https://github.com/emqx/emqx", {branch, "master"}, "apps/emqx"}}
+            , {emqx, {git_subdir, "https://github.com/qzhuyan/emqx", {branch, "dev/william/bump-quicer"}, "apps/emqx"}} %% @TODO change it back to emqx
```

Comment:
Does this need to be changed before merging?  Or in a following PR?

Reply:
yes. Follow-up PR is needed. we have a chicken/egg issue here. without this emqx branch, emqtt test cases could not be verified.